content/en/docs/tasks/administer-cluster/cluster-management.md

```diff
@@ -210,4 +208,4 @@ You can use `kubectl convert` command to convert config files between different
 kubectl convert -f pod.yaml --output-version v1
 ```
 
-For more options, please refer to the usage of [kubectl convert](/docs/reference/generated/kubectl/kubectl-commands/#convert) command.
+For more options, please refer to the usage of [kubectl convert](/docs/user-guide/kubectl/{{page.version}}/#convert) command.
```

Comment:
Please update this link (and any other links in the file) to use `/content/en/docs/` where you see `/docs/` now. This is due to the Hugo migration.